central/role/mapper/telemetry.go

```diff
@@ -0,0 +1,14 @@
+package mapper
+
+import (
+	"github.com/stackrox/rox/central/telemetry/centralclient"
+	"github.com/stackrox/rox/generated/storage"
+)
+
+// addUserToTenantGroup adds the given user to the central tenant group so that
+// such users could be segmented by tenant properties.
+func addUserToTenantGroup(user *storage.User) {
+	if cfg := centralclient.InstanceConfig(); cfg.Enabled() {
+		cfg.Telemeter().Group(cfg.GroupID, cfg.HashUserID(user.GetId(), user.GetAuthProviderId()), nil)
+	}
```

Comment:
What is the motivation for extracting these three lines into a separate function in a separate file? Do you envision its reuse and more telemetry-related functions in this file?

Reply:
The motivation is to extract the lower level telemetry code out of the role mapper logic.
I envision to continue this practice of keeping as less inline telemetry logic as possible. Reuse if possible, but that was not the intention here.